postgresql_extensions/tests/portal_corp.rs

```diff
@@ -0,0 +1,36 @@
+#[cfg(not(all(target_os = "macos", target_arch = "x86_64")))]
```

Comment:
Is there any existing testing infrastructure to check if the extension's operations work? It could be the simplest use case like:
```sql
CREATE EXTENSION vector;
CREATE TABLE items (id bigserial PRIMARY KEY, embedding vector(3));
INSERT INTO items (embedding) VALUES ('[1,2,3]'), ('[4,5,6]');
SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
```
I'd be happy to contribute code for this, just wanted to ask if there is existing files made for something like this.

Reply:
I think expanding this test would be ideal.  For the pgvecto.rs extension, I added a [test to the example](https://github.com/theseus-rs/postgresql-embedded/blob/main/examples/vector_extension/src/main.rs) to ensure that it is working.

Reply:
@spikecodes please feel free to submit a PR for any changes you'd like to see!